src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.CodeAnalysis.CSharp.ExtractMethod
 {
     internal partial class CSharpMethodExtractor
     {
-        private abstract partial class CSharpCodeGenerator : CodeGenerator<StatementSyntax, ExpressionSyntax, SyntaxNode, CSharpCodeGenerationOptions>
+        private abstract partial class CSharpCodeGenerator : CodeGenerator<StatementSyntax, SyntaxNode, CSharpCodeGenerationOptions>
```

Comment:
ExpressionSyntax is now inherited by us sublcassing `MethodExtractor<... ExpressionSyntax, ... >`

I couldn't do the same with StatementSyntax because in VB there are two types of statements used (StatementSyntax and ExecutableStatementSyntax) and vb wants to manage both separately.